docs/nodes/docker-images.mdx

```diff
@@ -270,3 +270,32 @@ you can refer to the
 </TabItem>
 </Tabs>
 ````
+
+## Persistent storage
+
+If you delete a container that you started above, all data will be lost.
+To avoid this, you can mount a volume to the container.
+This will allow you to persist data even after the container is deleted.
+
+Before mounting a volume, you will need to set permissions for
+the user on the host machine by running:
+
+```bash
+sudo chown 10001:10001 ./host/path
```

Comment:
This fails if the specified path does not exist. The user needs to create the folder first.